src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -115,68 +141,196 @@ module _ {_~_ : REL (List A) B ℓ} where
     (x~y ∷ (All.map All.head pxss)) ∷
     All-swap (x~ys ∷ (All.map All.tail pxss))
 
+------------------------------------------------------------------------
+-- Defining properties of lookup and _[_]=_
+--
+-- pxs [ i ]= px  if and only if  lookup pxs i = px.
+
+module _ {a p} {A : Set a} {P : A → Set p} where
+
```

Comment:
We have `variable`s for `a` and `A` so we can drop some of these parameters.

Reply:
and `p`